src/System.Private.CoreLib/src/System/Diagnostics/Contracts/ContractsBCL.cs

```diff
@@ -200,7 +200,8 @@ public void SetUnwind()
     [Serializable]
     [System.Runtime.CompilerServices.TypeForwardedFrom("mscorlib, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089")]
     [SuppressMessage("Microsoft.Design", "CA1064:ExceptionsShouldBePublic")]
```

Comment:
You can also delete this SupressMessage because of the type is public now...

Reply:
done